### DISCUSSION TITLE
performance issues adding a word from Emacs' eglot LSP mode

### DISCUSSION THREAD

**Discussion by anarcat** [Automattic/harper#185] [General]
In https://github.com/elijah-potter/harper/discussions/170, I mentioned that I suffered delays when adding words to the dictionary, and said I would investigate a little further to see if I could find error messages.

So I checked, and couldn't. I will try to turn up the debugging in eglot (but that affects performance!) but I wanted to open a separate thread for this to keep #170 alone on this.

For now, what I'm seeing is 1-2s delays. It's kind of hard to get a precise metric on this, I would need to do a screen recording and do a frame check, but from wall clock counting, I can say, mentally "mili-one, mili-two" and the thing completes about when i start the second "mili", which is about 1-2s, i think.

**Comment by elijah-potter** [comment]
Sorry it took so long for me to get back to you on this. @anarcat, do you still see performance problems in Emacs?
